daal4py/sklearn/linear_model/_logistic_path_0_22.py

```diff
@@ -1012,29 +1009,176 @@ def daal4py_predict(self, X, resultsToEvaluate):
     raise ValueError('resultsToEvaluate must be in [computeClassLabels, \
                      computeClassProbabilities, computeClassLogProbabilities]')
 
+def daal4py_fit(self, X, y, sample_weight=None):
```

Comment:
not critical, but it is better to name it with the prefix daal4py, what will call directly daal4py alforithms

Reply:
daal4py_fit is called only directly from LogisticRegression class